sdk/storage/test/test_base.cpp

```diff
@@ -92,22 +92,34 @@ namespace Azure { namespace Storage { namespace Test {
     return charset[distribution(random_generator)];
   }
 
-  std::string RandomString()
+  std::string RandomString(size_t size)
   {
     std::string str;
-    str.resize(10);
+    str.resize(size);
     std::generate(str.begin(), str.end(), random_char);
     return str;
   }
 
-  std::string LowercaseRandomString()
+  std::string LowercaseRandomString(size_t size)
   {
-    auto str = RandomString();
+    auto str = RandomString(size);
     std::transform(
         str.begin(), str.end(), str.begin(), [](unsigned char c) { return char(std::tolower(c)); });
     return str;
   }
 
+  std::map<std::string, std::string> RandomMetadata(size_t size)
```

Comment:
std::size_t

Reply:
`size_t` is more concise. We can bring this up in the next meeting.